src/Provider/GoogleMaps/GoogleMaps.php

```diff
@@ -304,6 +309,20 @@ private function updateAddressComponent(AddressBuilder $builder, string $type, $
                 $builder->addAdminLevel(intval(substr($type, -1)), $values->long_name, $values->short_name);
                 break;
 
+            case 'sublocality_level_1':
+            case 'sublocality_level_2':
+            case 'sublocality_level_3':
+            case 'sublocality_level_4':
+            case 'sublocality_level_5':
+                $newSubLocalityLevel[] = [
+                    'level' => intval(substr($type, -1)),
+                    'name' => $values->long_name,
+                    'code' => $values->short_name,
+                ];
+                $subLocalityLevel = ($builder->hasValue('subLocalityLevel')) ? $builder->getValue('subLocalityLevel') : [];
+                $builder->setValue('subLocalityLevel', array_merge($newSubLocalityLevel, $subLocalityLevel));
```

Comment:
Well `$subLocalityLevel` array is defined only in this function, and doesn't remember it's previous value.. I could add it as class property, and merge wont be needed.